src/applications/simple-forms/21-0966/tests/pages/phoneAndEmailAddress.unit.spec.jsx

```diff
@@ -0,0 +1,30 @@
+import {
+  testNumberOfErrorsOnSubmitForWebComponents,
+  testNumberOfWebComponentFields,
+} from '../../../shared/tests/pages/pageTests.spec';
+import formConfig from '../../config/form';
+
+const {
+  schema,
+  uiSchema,
+} = formConfig.chapters.contactInformationChapter.pages.phoneAndEmailAddress;
+
+const pageTitle = '';
```

Comment:
I seem to have passing tests even though this is an empty string. I haven't looked into how it's used but I guess its OK? 🤷

Reply:
change request: if tests fail, that will help you know what failed, so would be good to fill out